### ISSUE TITLE
Move request cart close button outside of table caption

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1742]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

Request cart

### Description of error / warning

Review of structural elements.

The close button is a control mechanism for the panel, not part of the table description

### WCAG success criterion

[1.3.1 Info and Relationships (Level A)](https://webaim.org/standards/wcag/checklist#sc1.3.1)

### WCAG conformance level

A

### Recommended fix

Move the close button up out of the table element

After this change is made, navigation of the cart should be tested with VoiceOver. Currently the aria-label is not read aloud for this close button.

### Tool used to find the problem

wave

### Screenshot

```html
<div data-v-0f02435f="" class="panel">
<table data-v-0f02435f="" class="lux-data-table fixed-header">
<caption data-v-0f02435f="">
<button data-v-5ed6c2fd="" data-v-0f02435f="" type="button" class="lux-button text medium denied-button" width="26px" aria-labelledby="closeCart">
```
